synapse/lib/lmdbslab.py

```diff
@@ -971,7 +975,8 @@ def _saveOptsFile(self):
         if self.readonly:
             return
 
-        opts = {}
+        opts = {'int_encoding': s_common.int64en_native(1)}
```

Comment:
Is this trying to save binary data into a YAML file or am i missing something?

Reply:
Yeah it is saving it to the yaml opts file. It ends up base64 encoded in there.

Reply:
Could we change this to something like `arch_endian` and the string `little` or `big` for clarify?